content/response-to-reviewers.md

```diff
@@ -134,22 +166,20 @@ Please see discussion at <https://github.com/greenelab/iscb-diversity-manuscript
 
 > p. 9, Figure 4: The same point can be made here as for Figure 2.  It would be nice to see a statistical analysis and a p-value or two to justify claims beyond appealing to figures.
 
-During revision, we have added overall statistics overall statistics for gender comparison between authors and iscb honorees.
+During revision, we have added overall statistics for gender comparison between authors and iscb honorees.
 Please see discussion at <https://github.com/greenelab/iscb-diversity-manuscript/issues/56>
 
 > p. 9, Figure 4: The term "other categories" is used but these are never really defined (see previous point).  It would be good to do so, perhaps even in a supplement.
 
-In the revised version of the manuscript, we spefidied which regions category O contains in the caption of this figure.
+In the revised version of the manuscript, we specified which regions category O contains in the caption of this figure.
 Please see discussion at <https://github.com/greenelab/iscb-diversity-manuscript/issues/57>.
 
 ## Review 4
 
 > This manuscript looks at the representation issues when honoring scientists in the field of computational biology by a computational biology society (ISCB). It establishes a background distribution based on the authors of relevant journal publications and then studies subgroup representation of keynotes at three of the ISCB-affiliated conferences (ISMB, RECOMB, PSB) and ISCB fellows against that distribution. It analyzes predicted gender, nationalities, ethnicities, and race of honorees. There are several findings but the major and only stated conclusion in Abstract is that there is recognition overrepresentation of white scientists and underrepresentation of non-white scientists.
 > 
 > This topic is out of scope for the proceedings of ISMB, at least by how I interpret the scope. It has its place in the published literature (I recommend with major revisions), but there is no methodology for or the analysis of molecular and biological data. Instead it is a study about the authors of such papers and broadly speaking it characterizes the recognition and reward system for their contributions to the field.
-
-Discussion at <https://github.com/greenelab/iscb-diversity-manuscript/issues/58>
-
```

Comment:
We don't want to link to issue? I feel like if we have an issue we might as well link in case readers want to comment more on the point.

Reply:
I thought this comment was irrelevant but OK I can add back in.